tools/export/exporters.py

```diff
@@ -196,6 +197,7 @@ def scan_and_copy_resources(self, prj_paths, trg_path, relative=False):
             self.config_header = self.toolchain.MBED_CONFIG_FILE_NAME
             config.get_config_data_header(join(trg_path, self.config_header))
             self.config_macros = []
+            self.resources.inc_dirs.append(dirname(self.config_header))
```

Comment:
I don't think this does anything useful. 3 lines above (line 197) we set `config_header` to the **name** of the config header file, not to its path. So its **dirname** will always be empty.